src/e2e/java/teammates/e2e/pageobjects/InstructorHomePage.java

```diff
@@ -119,14 +119,14 @@ public void deleteSession(int courseTabIndex, int sessionIndex) {
     public void archiveCourse(int courseTabIndex) {
         WebElement courseTab = getCourseTab(courseTabIndex);
         click(courseTab.findElement(By.className("btn-course")));
-        clickAndConfirm(courseTab.findElement(By.className("btn-archive-course")));
+        clickAndConfirm(browser.driver.findElement(By.cssSelector("body > div > div >a.btn-archive-course.btn")));
         waitUntilAnimationFinish();
     }
 
     public void deleteCourse(int courseTabIndex) {
         WebElement courseTab = getCourseTab(courseTabIndex);
         click(courseTab.findElement(By.className("btn-course")));
-        clickAndConfirm(courseTab.findElement(By.className("btn-delete-course")));
+        clickAndConfirm(browser.driver.findElement(By.cssSelector("body > div > div > a.btn-delete-course.btn")));
```

Comment:
Same as above - can this be shortened to `body > div > div > .btn-delete-course`? Seems a bit more readable to me, and conveys the same meaning.